middleware/middleware-1.3.30.md

```diff
@@ -0,0 +1,40 @@
+---
+slug: /release-notes/middleware/1.3.30
+title: Version 1.3.30
+---
+
+# fiskaltrust.Middleware 1.3.30 (Germany)
+_March 17, 2022_
+
+In this version of the Middleware, we've added the possibility to declare a 'Master Queue' if there are multiple Queues connected to one TSE, in order to avoid long timeouts during daily closings and have one tar file export for all queues.
+
+:::caution
+
+Version 1.3 of the Middleware is meant for the German market only, customers in Austria and France should continue to use version 1.2. We will unify these experiences in an upcoming version.
+
+:::
+
+## Feature: Possibility to enable a 'Master Queue' for TAR file export
+
+When setting up the parameter `EnableTarFileExport`, there is a possibility to select one queue which will be responsible for the daily closing and Tar file export.
+
+## Bug Fix: CryptoVision execption about data fragmentation on transport layer
+
+The source of the exception about data fragmentation on transport layer has been identified and corrected.
+
+## Bug Fix: Cryptovision timeout when downloading TAR file
+
+We fixed a bug happening during Tar files export with a TSE failed with a timeout error.
```

Comment:
Maybe add: "This bug only occurred when very large TAR files were exported, and only affected a small subset of users."